apps/portfolio/docs/plan/og-image-source.html

```
@@ -0,0 +1,77 @@
+<style>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)